packages/manager/modules/dbaas-logs/src/logs/detail/encryption-keys/home/home.scss

```diff
@@ -0,0 +1,6 @@
+.ellipsis {
+  max-width: 250px;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  display: inline-block;
+}
```

Comment:
**nitpick:** Avoid custom css if it can be achieved using bootstrap utilities.

Reply:
Removed